### PR TITLE
Order view page reference

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Describe order view page located in `Sell -> Orders -> Orders -> View`.
| Fixed ticket? |

### CODE REVIEW

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Refund - read about refunds [here]({{< relref "./refunds" >}}).
+- Next/Previous - allows to quickly jump to next or previous order in the list (simple `href` buttons redirecting to another `View order` page).
+
+#### Customer block
+{{< figure src="./img/customer-block.png" title="Order customer block" >}}
+
+Contains information like customer `email`, `addresses`, `orders count`, `total spent sum`. Click the `View full details` link to see more information about the customer (it redirects to `Customers -> Customers -> View customer` page). This block also allows following actions:
+- Editing and selecting new customer shipping and/or invoice addresses (both of these actions opens a modal with a `Customers -> Addresses -> Edit` page inside an `iframe` using the `Lite display` mode of the Back Office). See [OrderController::changeCustomerAddressAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
```
Comment (matks) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:38]: Second time we mention Lite Display.

Would it be possible to submit another PR to explain it 😉  ?
It's a TODO-item for docs https://github.com/PrestaShop/docs/issues/231 😄

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+{{% notice %}}
+It is possible to have multiple invoices related to same order, therefore when editing a product or adding a discount you can select which invoice to use.
+{{< figure src="./img/order-invoice-select.png" title="Order invoice select" >}}
+{{% /notice %}}
+
+#### Messages block
+{{< figure src="./img/order-messages-bo.png" title="Order messages in Back Office" >}}
+
+This block shows order messages. These messages can be visible for the customer in Front Office order details or can stay hidden - that depends on the checkbox `Display to customer`.
```
Comment (matks) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:65]:
```suggestion
This block shows order messages (that rely on OrderMessage objectModel). These messages can be visible for the customer in Front Office order details or can stay hidden - that depends on the checkbox `Display to customer`. Messages come from
```

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+#### History block
+{{< figure src="./img/history-block.png" title="Order history block" >}}
+
+This block doesn't have an actual title like others, but let's just call it "History block" for now - it's the one containing following tabs:
+- Status - you can see transitions of this order statuses and update the status, you can also add an Order note in this tab. (`Order->note`). See [OrderController::setInternalNoteAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Documents - this tab contains invoices list.
+- Carriers - contains carriers information. You can change the current carrier and add a tracking number. See [OrderController::updateShippingAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Merchandise returns - information about refunds history.
```
Comment (matks) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:76]: Maybe we could quickly explain how modules can customize this part? Related to page https://devdocs.prestashop.com/1.7/modules/sample-modules/order-pages-new-hooks/

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+#### Actions block
+{{< figure src="./img/actions-block.png" title="Order actions block" >}}
+
+This block contains following actions:
+- Status update - new order status can be selected from a dropdown. See [OrderController::updateStatusAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- View invoice - generates invoice PDF. See [OrderController::generateInvoicePdfAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Print order - prepares and shows modified page layout for printing. This action is performed by javascript in [admin-dev/themes/new-theme/js/pages/order/view](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/admin-dev/themes/new-theme/js/pages/order/view)
+- Refund - read about refunds [here]({{< relref "./refunds" >}}).
+- Next/Previous - allows to quickly jump to next or previous order in the list (simple `href` buttons redirecting to another `View order` page).
+
```
Comment (matks) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:33]: Maybe we can quickly mention that modules can add buttons using `actionGetAdminOrderButtons ` hook (see https://devdocs.prestashop.com/1.7/modules/sample-modules/order-pages-new-hooks/)

**Review by matks — APPROVED** [review]
Approved for me 👍 

I added suggestions but they are not blocking for merge

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md (hunk 4 above)
Comment (zuk3975) [reply src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:33]: I've added the link to new hooks above, so I don't think its worth mentioning it in every block now.

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+---
+
+# Order View Page
+
+{{% notice tip %}}
+**New hooks** are available for this page. Check it out [here](./../../../../../modules/sample-modules/order-pages-new-hooks) {{< minver v="1.7.7.0" >}}
+{{% /notice %}}
+
+Page can be reached by visiting `Sell -> Orders -> Orders -> View (grid row action)`. It allows the Back Office user to view the details of selected order and edit it. The **related code can be found in following locations**:
+- Main **twig template** [src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig).
```
Comment (Progi1984) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:13]:
```suggestion
- Main **twig template** [src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig](https://github.com/PrestaShop/PrestaShop/tree/develop/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig).
```

File: src/content/1.7/development/page-reference/back-office/order/view-order/_index.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+- **Javascript** ([pre-compiled]({{< relref "compile-assets" >}})) [admin-dev/themes/new-theme/js/pages/order/view](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/admin-dev/themes/new-theme/js/pages/order/view).
+- **OrderController** [src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php) (other domain controllers are used as well, those will be mentioned in related block references bellow) 
+- **Commands** & **Queries** at [src/Core/Domain/Order](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Core/Domain/Order) in `Command`, `Query` directories.
+- Most **handlers** or **domain services implementations** can be found in `Adapter` namespace [src/Adapter/Order](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Adapter/Order)
```
Comment (Progi1984) [inline src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:17]:
```suggestion
- **Javascript** ([pre-compiled]({{< relref "compile-assets" >}})) [admin-dev/themes/new-theme/js/pages/order/view](https://github.com/PrestaShop/PrestaShop/tree/develop/admin-dev/themes/new-theme/js/pages/order/view).
- **OrderController** [src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php) (other domain controllers are used as well, those will be mentioned in related block references bellow) 
- **Commands** & **Queries** at [src/Core/Domain/Order](https://github.com/PrestaShop/PrestaShop/tree/develop/src/Core/Domain/Order) in `Command`, `Query` directories.
- Most **handlers** or **domain services implementations** can be found in `Adapter` namespace [src/Adapter/Order](https://github.com/PrestaShop/PrestaShop/tree/develop/src/Adapter/Order)
```

Comment (Progi1984) [reply src/content/1.7/development/page-reference/back-office/order/view-order/_index.md:17]: Please target all links to `develop`.

**Comment by zuk3975** [comment]
thanks for feedback, resolved!

**Comment by matks** [comment]
Thank you @zuk3975
